app/models/grant.rb

```diff
@@ -1,28 +1,30 @@
 class Grant < ActiveRecord::Base
   has_many :grants_voters
+  has_many :grant_submissions
 
-  validates :name, :presence => true
-  validates :max_funding_dollars, :presence => true, :numericality => {:greater_than => 0, :only_integer => true}
-  validates :submit_start, :presence => true
-  validates :submit_end, :presence => true
-  validates :vote_start, :presence => true
-  validates :vote_end, :presence => true
-  validates :meeting_one, :presence => true
-  validates :meeting_two, :presence => true
+  validates :name, presence: true
+  validates :max_funding_dollars, presence: true, numericality: { greater_than: 0, only_integer: true}
+  validates :submit_start,  presence: true
```

Comment:
`blah: :foo` is the newer syntax. I will likely add `rubocop` as a style linter once this project is in better shape and it will warn when using the old syntax. It's a small style thing like 80 character line limits.